src/test/java/org/kiwiproject/elk/ElkAppenderFactoryTest.java

```diff
@@ -0,0 +1,100 @@
+package org.kiwiproject.elk;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AsyncAppenderBase;
+import io.dropwizard.logging.async.AsyncLoggingEventAppenderFactory;
+import io.dropwizard.logging.filter.ThresholdLevelFilterFactory;
+import net.logstash.logback.appender.LogstashUdpSocketAppender;
+import net.logstash.logback.layout.LogstashLayout;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+@DisplayName("ElkAppenderFactory")
+class ElkAppenderFactoryTest {
+    private static final String APP_NAME = "aTestApp";
+
+    @Test
+    @SuppressWarnings("java:S5778")
```

Comment:
Remove all these warning suppressions and let's instead do it in the Sonar UI. That's what I did for Kiwi to avoid cluttering up the code with all these.

Reply:
You can also "fix" the problem it is whining about by extracting those as variables _outside_ the `assertThatThrownBy`. And since several of these tests use the same things, maybe just put them, as instance fields and initialize them in the `@BeforeEach` ???

Reply:
Done